deploy/complete/helm-chart/provision/templates/service-binding-global.yaml

```diff
@@ -36,4 +36,6 @@ metadata:
 spec:
   instanceRef:
     name: {{ .Values.global.osb.instanceName }}-objectstorage
+  parameters:
+    generatePreAuth: true
```

Comment:
@junior  this is the reason for the empty binding. Note too that we have an issue where the `serviceinstance` cannot be deleted due to objects in the bucket. Let's discuss a solution for that

Reply:
The TF should track each object inside the bucket to safely delete the objects with TF Destroy. This affects the Always-Free Terraform scripts. The Binding be created by Helm is different, on the Complete Terraform Scripts, which is handled by the terraform helm provider.

Reply:
Include helm changes on the basic TF PR confused a lit bit. Why cannot delete objects from TF?

Reply:
The TF stack now does track each bucket object in the config. The helm chart contained no PAR for image asset deployment, hence this change. But, once images are added (from K8s, or otherwise manually, or through an admin interface) then the bucket cannot be deleted.

Reply:
If the images are tracked within the helm provider to the tf state, will be deleted. 

Will not be deleted if new products with new images (or images changes for original products) be included. This is valid for both, basic and complete.